src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import 'bulma';
+import '@fortawesome/fontawesome-free/css/all.css';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+import './Header.scss';
+
+export const Header: React.FC = () => (
+  <nav
+    className="navbar is-primary"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="navbar-brand">
+      <a className="navbar-item" href="./">
+        <img
+          src="https://static.codedojo.ru/assets/images/topics/react.png"
+          alt="logo"
+        />
+      </a>
+    </div>
+
+    <div id="navbarBasicExample" className="navbar-menu">
+      <div className="navbar-start">
+        <NavLink
+          to="/"
+          className={({ isActive }) => classNames('navbar-item', {
+            'navbar-item--active': isActive,
```

Comment:
```suggestion
          className={classNames('navbar-item', {
            'navbar-item--active': isActive,
```
А таким чином хіба не спрацює? 🤔

Reply:
Думав що ні) не пробував )